torch/csrc/jit/passes/onnx.cpp

```diff
@@ -15,7 +15,7 @@
 #include <torch/csrc/jit/python/python_ir.h>
 #include <torch/csrc/utils/pybind.h>
 #include <sstream>
-#include <unordered_map>
+#include <unordered_set>
```

Comment:
unordered_map is not longer used, but this file still uses unordered_set which relied on a transitive include